pkg/dao/repository_test.go

```diff
@@ -515,6 +516,25 @@ func (suite *RepositorySuite) TestListFilterMultipleVersions() {
 	assert.Equal(t, int64(quantity), count)
 }
 
+func (suite *RepositorySuite) TestSavePublicUrls() {
+	t := suite.T()
+	repoUrls := []string{"https://somepublicRepo.com/", "https://anotherpublicRepo.com/"}
+	err := GetRepositoryDao(suite.tx).SavePublicRepos(repoUrls)
+
+	require.NoError(t, err)
+	repo := models.Repository{}
+	result := suite.tx.Where("url = ?", repoUrls[0]).Find(&repo)
+
+	assert.Nil(t, result.Error)
+	var count int64
+	suite.tx.Model(&repo).Count(&count)
+	assert.Equal(t, int64(2), count)
+
+	err = GetRepositoryDao(suite.tx).SavePublicRepos(repoUrls)
+	assert.Nil(t, err)
```

Comment:
```suggestion
	assert.NoError(t, err)
```
It adds more semantic to the statement.